src/server/game/Entities/Unit/Unit.cpp

```diff
@@ -2106,6 +2106,12 @@ void Unit::AttackerStateUpdate(Unit* victim, WeaponAttackType attType, bool extr
         DamageInfo dmgInfo(damageInfo);
         ProcSkillsAndAuras(damageInfo.target, damageInfo.procAttacker, damageInfo.procVictim, PROC_SPELL_TYPE_NONE, PROC_SPELL_PHASE_NONE, dmgInfo.GetHitMask(), nullptr, &dmgInfo, nullptr);
 
+        if (dmgInfo.GetHitMask() & PROC_HIT_CRITICAL) // @todo is there a better place to put this hack? there doesn't seem to be any aura applied on talent
+            if (Pet* pet = ToPet())
+                if (pet->HasSpell(53508)) // Wolverine Bite
+                    pet->AddComboPoints(damageInfo.target, 1);
```

Comment:
It has `SPELL_ATTR4_TRIGGER_ACTIVATE` and `SPELL_ATTR1_UNK27`, no idea if either of those is interesting.